turbo/execution/eth1/eth1_chain_reader.go/chain_reader.go

```diff
@@ -281,22 +279,29 @@ func (c ChainReaderWriterEth1) InsertBlocksAndWait(ctx context.Context, blocks [
 	if err != nil {
 		return err
 	}
-	retryInterval := time.NewTicker(retryTimeout)
-	defer retryInterval.Stop()
+
+	// limit the number of retries
+	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	defer cancel()
 
 	for response.Result == execution.ExecutionStatus_Busy {
+		const retryDelay = 100 * time.Millisecond
+		delayTimer := time.NewTimer(retryDelay)
```

Comment:
to not mess with `.Stop` method try `time.After`

Reply:
:+1: done